code/modules/antagonists/blob/blob/blobs/shield.dm

```diff
@@ -45,6 +45,8 @@
 	explosion_block = 2
 
 /obj/structure/blob/shield/reflective/handle_ricochet(obj/item/projectile/P)
+	if(istype(P,/obj/item/projectile/beam/beam_rifle/hitscan/aiming_beam))

```

Comment:
I hate using an istype to solve this, a more generic solution should be used

Reply:
this is the only item that has this interaction or I would I have done it from the getgo, but I can make it more generic if you REALLY want :P

Reply:
If you have one thing that uses a functionality like this you're going to want more